### ISSUE TITLE
DatePicker

### ISSUE THREAD

**Issue by balloonhotair** [gjr80/weewx-highcharts#7]
Hi Gary,
I have added a jquery datepicker to plots.js to use on a range selector in the weewx highcharts extension. Do you want me to send it to you? It is only 4 lines in plots.js and two lines in the parent page to load the javascript and css.
Thanks Jerry

**Comment by balloonhotair** [comment]
[plots.js.txt](https://github.com/gjr80/weewx-highcharts/files/3262297/plots.js.txt)
[convert_units.js.txt](https://github.com/gjr80/weewx-highcharts/files/3262298/convert_units.js.txt)
[test.html.txt](https://github.com/gjr80/weewx-highcharts/files/3262299/test.html.txt)

**Comment by balloonhotair** [comment]
Hi Gary,
I attached the files and a test html file to this issue.  In addition to adding a date picker to plots I also added a initial approach to allowing units to be changed at display time. I only modified the barometer display data in the weekly path. If you are ok with this approach I will do the rest of the fields and the yearly path. The file convert_units.js is the support file for doing the different units conversions.
